src/hotspot/share/oops/fieldStreams.hpp

```diff
@@ -25,7 +25,7 @@
 #ifndef SHARE_OOPS_FIELDSTREAMS_HPP
 #define SHARE_OOPS_FIELDSTREAMS_HPP
 
-#include "oops/instanceKlass.hpp"
+#include "oops/instanceKlass.inline.hpp"
```

Comment:
including .inline.hpp from .hpp is against the guidelines.  You should move things and include instanceKlass.inline.hpp in fieldStreams.inline.hpp instead.